packages/@aws-cdk/cloudformation-include/lib/cfn-include.ts

```diff
@@ -13,6 +13,11 @@ export interface CfnIncludeProps {
    * Both JSON and YAML template formats are supported.
    */
   readonly templateFile: string;
+
+  /**
+   * Specifies the template files that define nested stacks that should be included.
```

Comment:
We need more docs here. For example:

1. What happens when you have a nested stack in your template, but don't include it here.
2. What if you include an ID here that does not exist in the template.
3. What happens here if you include an ID here that exists in the template, but is not a nested stack.